tools/src/h5stat/h5stat.c

```diff
@@ -1086,7 +1086,7 @@ print_file_info(const iter_t *iter)
     printf("\t# of unique links: %lu\n", iter->uniq_links);
     printf("\t# of unique other: %lu\n", iter->uniq_others);
     printf("\tMax. # of links to object: %lu\n", iter->max_links);
-    fprintf(stdout, "\tMax. # of objects in group: %" PRIuHSIZE "\n", iter->max_fanout);
```

Comment:
Seems like all the other prints above should be consistent here so that you don't have split output when `rawoutstream` is something other than `stdout` (like a file). I imagine there are probably many other printf, putc, etc. calls like this.

Reply:
That is one of the tools that is out of step and needs to fixed. I'll see if I can convert those printfs.